### PR TITLE
Fix: Add missing number formats to language strings

### LINKED ISSUE #2214: Incorrect power level strings
Incorrect power level strings. In German language.

power1

power2

### PR DESCRIPTION
**Pull request by xezon**
* Fixes #2214

This change adds missing number formats to language strings. And also fixes minor white space issues around numbers.

Affects

* TOOLTIP:GameInfoPlayer for German, Chinese
* CONTROLBAR:PowerDescription for German
* ~~MOTD:NumPlayersHeading for all native languages~~

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/generals.str [hunk 1]
```diff
@@ -37358,16 +37362,18 @@ RU: "ПОБЕДИТЕЛЬ ПРОШЛОЙ НЕДЕЛИ"
 AR: "ﻲﺿﺎﻤﻟا عﻮﺒﺳﻷا وﺰﺋﺎﻓ"
 END
 
+// Patch104p @fix xezon 07/08/2023 Adds player numbers to all strings. Copied from Generals.
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/generals.str]: Does it work?

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/generals.str]: Not sure. I am no longer able to connect to cnc online.

shot_20230810_184956_1

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/generals.str]: Looks like so:

shot_20230811_144609_2

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/generals.str]: Sentence removed.

Comment (commy2) [reply Patch104pZH/GameFilesEdited/Data/generals.str]: 162 million ZH players 👍

**Comment by xezon** [comment]
Documentation added. Looks good.

**Pull request merged**